pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 100
+target-version = ['py36']
```

Comment:
do we want the pyporject.toml for anything else? If we only need it for black, then we already have a black-config.toml, it's just not referenced in many places because there's not much in there, but you'd use it by doing

`black --config=.black-config.toml`